deploying.md

```diff
@@ -0,0 +1,37 @@
+# Deploying
+There are multiple ways of deploying the defrag API.
+## Using Podman
+Using Podman is probably the easiest method.
+
+### Using prebuilt containers
+1. Create a .env file as described in [configuration.md](configuration.md).
+2. Make sure you have Podman installed: `sudo zypper install podman`
+3. Run the container: `podman run --env-file /path/to/env/file -p 80:8000 registry.opensuse.org/home/karatekhd/defrag/containers/defrag:latest`. Replace the `/path/to/env/file` with the actual path tp your env file.
```

Comment:
Also this will trigger change to `defrag/__init__.py`, won't it? Do we still need the `config` parser after this change?

Reply:
No, it won't. This parameter does the same creating a `.env` file does. It just passes it to the container so that it can read the variables specified in the env file.

Reply:
We still need the config parser though.

Reply:
Alright then.